src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/AssetsComputingHelper.cs

```diff
@@ -107,7 +107,7 @@ public static string GetCandidateRelativePath(ITaskItem candidate, bool fingerpr
         {
             fileName = Path.GetFileNameWithoutExtension(destinationSubPath);
             extension = Path.GetExtension(destinationSubPath);
-            subPath = destinationSubPath.Substring(fileName.Length + extension.Length);
+            subPath = destinationSubPath.Substring(0, destinationSubPath.Length - (fileName.Length + extension.Length)); ;
```

Comment:
that's fine. Why not `Path.GetDirectoryName(destinationSubPath);`?

Reply:
Sounds good!